app/src/main/java/com/google/android/diskusage/ui/filemanagers/DocumentsUIFilemanager.kt

```diff
@@ -0,0 +1,33 @@
+package com.google.android.diskusage.ui.filemanagers
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import android.provider.DocumentsContract
+import android.util.Log
+import com.google.android.diskusage.filesystem.entity.FileSystemEntry
+
+class DocumentsUIFilemanager(mEntry: FileSystemEntry, mActivity: Activity) : ExternalFileManager(mEntry, mActivity) {
+
+    override fun getFilemanagerIntent(): Intent {
+        // Documentation for the reason behind the replacements
+        // https://stackoverflow.com/a/75299820
+        var path = mEntry.absolutePath()
```

Comment:
Looks scary. I think it will be nightmare to maintain and can fail in all possible ways on different versions of android / different manufactures.

Reply:
Yeah, i dont really like that "solution" too, at least not without finding a better documentation for it.

Do you have any idea why the original "Default Filemanager" doesn't work in the emulator/aosp?